artifacts/linting_rules/lint_function/camara-reserved-words.js

```diff
@@ -0,0 +1,95 @@
+const reservedWords = [
+  'abstract',
+  'apiclient',
+  'apiexception',
+  'apiresponse',
+  'assert',
+  'boolean',
+  'break',
+  'byte',
+  'case',
+  'catch',
+  'char',
+  'class',
+  'configuration',
+  'const',
+  'continue',
+  'do',
+  'double',
+  'else',
+  'extends',
+  'file',
+  'final',
+  'finally',
+  'float',
+  'for',
+  'goto',
+  'if',
+  'implements',
+  'import',
+  'instanceof',
+  'int',
+  'interface',
+  'list',
+  'localdate',
+  'localreturntype',
+  'localtime',
+  'localvaraccept',
+  'localvaraccepts',
+  'localvarauthnames',
+  'localvarcollectionqueryparams',
+  'localvarcontenttype',
+  'localvarcontenttypes',
+  'localvarcookieparams',
+  'localvarformparams',
+  'localvarheaderparams',
+  'localvarpath',
+  'localvarpostbody',
+  'localvarqueryparams',
+  'long',
+  'native',
+  'new',
+  'null',
+  'object',
+  'offsetdatetime',
+  'package',
+  'private',
+  'protected',
+  'public',
```

Comment:
NOTE: In BlockchainPublicAddress the word "public" is part of the API semantic.
Just to comment here: good to keep as a traversal rule, but in blockchain this check will be explicitly ommitted